README.md

```diff
@@ -0,0 +1,39 @@
+# Workflow CA
+
+### Workflow status badges
+
+[![Deploy static content to Pages](https://github.com/maglus02/social-media-client/actions/workflows/pages.yml/badge.svg)](https://github.com/maglus02/social-media-client/actions/workflows/pages.yml)
+
+## Installation
+
+1. **Clone the repository:**
+   
+   ```sh
+   git clone https://github.com/maglus02/social-media-client.git
+   ```
+   
+2. **Navigate to the project directory:**
+   
+   ```sh
+   cd social-media-client
+   ```
+   
+3. **Install dependencies:**
+   
+   ```sh
+   npm install
+   ```
+   
+4. **Start the application**
+   
+   ```sh
+   npm start
+   ```
+   
+
+## Testing
+Run the following command to test Jest and Cypress:
```

Comment:
It would be nice to have instructions for all the testing operations that are possible